framework/wazuh/core/config/models/logging.py

```diff
@@ -0,0 +1,90 @@
+import logging
+
+from pydantic import BaseModel, Field
+from typing import Literal, List
+
+
+class LoggingConfig(BaseModel):
+    """Configuration for logging levels.
+
+    Parameters
+    ----------
+    level : Literal["info", "debug", "debug2"]
+        The logging level. Default is "info".
+    """
+    level: Literal["info", "debug", "debug2"] = "info"
```

Comment:
Create an enumerator for these values and the ones used in other classes

Reply:
Added in e4e54327c2b3a45cbe77a6f9c30cfd19dc9ce0b6.